packages/builder/src/pages/builder/portal/manage/users/[userId].svelte

```diff
@@ -0,0 +1,156 @@
+<script>
+  import { goto } from "@roxi/routify"
+  import {
+    ActionButton,
+    Button,
+    Layout,
+    Heading,
+    Body,
+    Divider,
+    Label,
+    Input,
+    Modal,
+    Table,
+    ModalContent,
+    notifications,
+  } from "@budibase/bbui"
+  import { fetchData } from "helpers"
+  import { users, apps } from "stores/portal"
+
+  import UpdateRolesModal from "./_components/UpdateRolesModal.svelte"
+
+  export let userId
+  let deleteUserModal
+  let editRolesModal
+
+  const roleSchema = {
+    name: { displayName: "App" },
+    roles: { type: "options" },
+  }
+
+  // Here we need to merge the Apps list and the roles response to get something that makes sense for the table
+  $: appList = $apps.map(app => ({ ...app, roles: ["READ"] }))
+  let selectedApp
+
+  const request = fetchData(`/api/admin/users/${userId}`)
+
+  async function deleteUser() {
+    const res = await users.del(userId)
+    if (res.message) {
+      notifications.success(`User ${$request?.data?.email} deleted.`)
+      $goto("./")
+    } else {
+      notifications.error("Failed to delete user.")
+    }
+  }
+
+  async function openUpdateRolesModal({ detail }) {
+    selectedApp = detail
+    editRolesModal.show()
+  }
+</script>
+
+<Layout noPadding gap="XS">
+  <div class="back">
+    <ActionButton on:click={() => $goto("./")} quiet size="S" icon="BackAndroid"
+      >Back to users</ActionButton
```

Comment:
I've seen my prettier do this weird new line thing too, but yours seems to be even more aggressive 😄 maybe a `format` would fix it? There's loads of these style of indentations in quite a few files.

Reply:
Yeah, no clue why it does this, it looks very ugly indeed! 😄